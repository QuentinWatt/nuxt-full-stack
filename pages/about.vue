<template>
  <main class="container mx-auto px-3 py-6">
    <h1>About us</h1>
  </main>
</template>
