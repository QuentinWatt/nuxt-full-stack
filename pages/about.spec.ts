import { expect, test } from "vitest";
import about from "./about.vue";
import { mountSuspended } from "@nuxt/test-utils/runtime";

test("It renders the H1 for the about page", async () => {
  const component = await mountSuspended(about);
  expect(component.text()).toContain("About us");
});
