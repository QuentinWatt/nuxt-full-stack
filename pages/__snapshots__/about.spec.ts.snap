// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`It matches the snapshot 1`] = `
"<main class="container mx-auto px-3 py-6">
  <h1>About us</h1>
</main>"
`;
